qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
Will this work? Remember that the issue is that this templates used to rely on the presence/absence of a message so when a general message was present it didn't work. Not sure what has to do with just maintenance, could you explain?   Also, not sure on the benefit of None vs. True/False, could you elaborate?